library/src/app/components/account/account-details/account-details.component.html

```diff
@@ -1,144 +1,247 @@
 <ng-container *ngIf="(isRecoverable$ | async) !== false; else error">
```

Comment:
This is a huge html file that looks to have separable components (e.g., transfer listing, trade listing, etc.). Can we break this into multiple partials so it's easier to review the HTML?